### PR TITLE
fetch bridge status

### PR DESCRIPTION
**Pull request by dks-mbta**

### CODE REVIEW

File: lib/notifications/bridge.ex [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+          bridge1_lowering_time: DateTime.t() | nil
+        }
+
+  @fetch_ms 60 * 1_000
+
+  @spec default_name() :: GenServer.name()
+  def default_name(), do: Notifications.Bridge
+
+  @spec start_link() :: GenServer.on_start()
+  @spec start_link(Keyword.t()) :: GenServer.on_start()
```
Comment (lemald) [inline lib/notifications/bridge.ex:23]: Double typespec (which, interestingly enough, doesn't seem to cause either `mix compile` or `mix dialyzer` to fail? That's weird.).

File: lib/notifications/bridge.ex [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+  def do_parse_response(response) do
+    # {"bridge":{"id":1,"name":"Chelsea St Bridge","bridgeStatusId":{"id":2,"status":"Lowered"}}}
+    #   or
+    # {"bridge":{"id":1,"name":"Chelsea St Bridge","bridgeStatusId":{"id":2,"status":"Raised","lift_estimate":{"estimate_time":"2019-09-26 06:47:05.0"}}}}
+
+    status = get_in(response, ["bridge", "bridgeStatusId", "status"])
+
+    lowering_time =
+      get_in(response, ["lift_estimate", "estimate_time"])
```
Comment (lemald) [inline lib/notifications/bridge.ex:109]: Can separate this out into another use of the pipe operator (`response |> get_in(...)`).

File: test/notifications/bridge_test.exs [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+          }
+        }
+      }
+
+      Bypass.expect(bypass, fn conn -> Plug.Conn.resp(conn, 200, Jason.encode!(json)) end)
+
+      assert handle_info(:update, state) == {:noreply, {"Lowered", nil}}
+    end
+
+    # , %{bypass: bypass} do
```
Comment (lemald) [inline test/notifications/bridge_test.exs:59]: Extraneous comment

**Comment by lemald** [comment]
Okay, a couple of comments but they're mostly minor syntactic things. The one additional bit is that you'll need to do some work to get the secrets from AWS Secrets Manager in prod, which you can do in `config/releases.exs`. There are already a few other examples in there.

**Comment by dks-mbta** [comment]
@lemald Ok, I made these changes. Let me know what you think.

**Review by lemald — APPROVED** [review]
Looks good to me! 👍

**Pull request merged**